### ISSUE TITLE
Tab components should show visible headings for all sections on mobile screen sizes

### ISSUE THREAD

**Issue by CharlotteDowns** [alphagov/govuk-design-system#2797]
This issue is from May 2023 external accessibility audit report.

**WCAG Reference:**
2.4.10 Section Headings (Level AAA)
[Understanding Section Headings](https://www.w3.org/WAI/WCAG21/Understanding/section-headings.html) | [How to Meet Section Headings](https://www.w3.org/WAI/WCAG21/quickref/#section-headings)

**Issue ID:**
DAC_Section_Headings_01

**URLs:**
[https://govuk-frontend-pr-3503.herokuapp.com/full-page-examples/start-page](https://govuk-frontend-pr-3503.herokuapp.com/full-page-examples/start-page)

## Screen shots
<img width="673" alt="Tabs view on desktop as tabs" src="https://github.com/alphagov/govuk-frontend/assets/56260216/593d2ed0-ec68-4346-9dee-5ee009c38099">
<img width="326" alt="Tabs view on mobile as in page links" src="https://github.com/alphagov/govuk-frontend/assets/56260216/6456551b-627c-429c-a4ff-81ce13f2a7a9">

Although when contained within tab panels, when viewing on a desktop, the sections of content were separable from one another and other content on the page. The lack of section headings in the alternative view did not enable users of screen reading assistive technologies to easily understand the purpose of the skip links.

This can also be confusing for other users when viewing the page with magnifying options as the skip links indicate the sections of content they will direct the user to, however, there are no headings to relate those links to the specific sections of content when viewing in this way.

## Current code ref(s)
```
<div class="govuk-tabs__panel" id="more-information" role="tabpanel" aria-labelledby="tab_more-information">
<p class="govuk-body">You’ll need a debit or credit card to use this service.</p>
<p class="govuk-body"><a class="govuk-link" href="#">It’s £9.50 cheaper</a> to apply for a passport online than by post.</p>
<p class="govuk-body">It should take around 6 weeks to get your first UK adult passport, but it can take longer.</p>
<p class="govuk-body">All other application types (for example, renewing a passport or getting a child passport) should take 3 weeks. It can take longer if more information is needed or your application hasn’t been filled out correctly.</p>
<p class="govuk-body">You should use a different service if you <a class="govuk-link" href="#">need your passport urgently</a>.</p>
</div>
```

## Screen-reader comments
> “With both VoiceOver and TalkBack, I found that the tabs instead of switching the content to a different view were in page links and were intended to move my focus to the relevant point of the page. 

> With VoiceOver when I selected the in-page links instead of moving to the indicated point on the page I was moved to the top of the page. This was disorientating as it was not initially clear where I had landed. I also became frustrated as it was time consuming to move back to the in-page links and then manually past the item to read the information. This functionality did not occur with TalkBack as focus moved to the intended point. 

> I then found that both sections were present on the page at once. This was confusing as no headings were available to advise me of where one section ended and the next began. It was therefore not clear whether I was viewing the ‘More info’ section or the ‘Other ways to apply’. 

> Ensuring that I can easily understand the difference between the sections and can move directly to them for example by placing a heading at the beginning of each section will prevent any confusion”

## Low vision user comments
>“While viewing this page with ‘windows high contrast BLACK’ and with ‘Chrome zoom x200%’ I found the ‘More information’ and ‘Other ways to apply’ tab links did not relate to any content on the page, and I found it difficult to see the different sections of content. Adding text to introduce the sections would help me to determine which information I am reading.”

## Solution
Consider adding headings to each tab panel to provide additional heading structure for users of screen reading assistive technologies, also providing them with an additional method of navigation.
These headings should also be provided when viewing in the alternative magnified and mobile views.
